linode/domainzonefile/framework_datasource_schema.go

```diff
@@ -0,0 +1,24 @@
+package domainzonefile
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var frameworkDatasourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"domain_id": schema.Int64Attribute{
+			Description: "The domain's ID.",
+			Required:    true,
+		},
+		"zone_file": schema.ListAttribute{
+			Description: "Lines of the zone file for the last rendered zone for this domain.",
+			Optional:    true,
```

Comment:
This field might need to be computed rather than optional 👍

Reply:
Fixed!